pipelines/controller/README.md

```diff
@@ -1,2 +1,80 @@
-Please see
-[this Wiki page](https://github.com/google/fhir-data-pipes/wiki/Try-out-the-FHIR-Pipelines-Controller).
+# FHIR Pipelines Controller
+
+The FHIR Pipelines Controller helps you schedule and manage the transformation
+of data from a [HAPI FHIR server](https://hapifhir.io/) to a collection of
+Apache Parquet files. It uses the FHIR Data Pipes JDBC pipeline to run either
+full or incremental transformations to a Parquet data warehouse.
+
+The FHIR Pipelines Controller only works with HAPI FHIR servers using Postgres.
```

Comment:
Only applies to JDBC mode and the note on PostgreSQL is not necessarily correct (although we have not tried), so:
```suggestion
The JDBC mode of FHIR Pipelines Controller only works with HAPI FHIR servers.
```